wearApp/src/main/java/com/surrus/peopleinspace/PersonDetailsScreen.kt

```diff
@@ -60,6 +60,9 @@ private fun PersonDetailsScreen(
     person: Assignment?,
     scrollState: ScrollState = rememberScrollState(),
 ) {
+    // Activate scrolling
+    LocalView.current.requestFocus()
```

Comment:
I suspect this should actually be

https://medium.com/google-developer-experts/focus-in-jetpack-compose-6584252257fe

```
val requester = FocusRequester()

    XXX(
        ...
        modifier = Modifier.focusRequester(requester)
    )

    // Request focus as a SideEffect (after the composition)
    SideEffect {
        if (item.isFocused) {
            requester.requestFocus()
        }
    }
```

Reply:
Is that something you want to change first or are you ok to merge what's in this right now?

Reply:
Merge now. I think its the same here. But  would be more impactful with say a form and the app moving the focus around actively.